README.md

```diff
@@ -2,3 +2,24 @@
 
 [![Build Status](https://travis-ci.org/dremio-hub/dremio-hbase-connector.svg?branch=master)](https://travis-ci.org/dremio-hub/dremio-hbase-connector)
 
+## Usage Instructions
+
```

Comment:
Consider adding an introduction to the procedure.  Typical style begins with the preposition 'to' and ends with a colon (:).  For example, "To add the HBase community connector to the Dremio list of sources:

Reply:
Done